stubs/jmespath/jmespath/exceptions.pyi

```diff
@@ -1,44 +1,41 @@
+from collections.abc import Sequence
 from typing import Any
 
 class JMESPathError(ValueError): ...
 
 class ParseError(JMESPathError):
-    lex_position: Any
-    token_value: Any
-    token_type: Any
-    msg: Any
-    expression: Any
-    def __init__(self, lex_position, token_value, token_type, msg=...) -> None: ...
+    lex_position: int
+    token_value: str
+    token_type: str
+    msg: str
+    expression: str | None
+    def __init__(self, lex_position: int, token_value: str, token_type: str, msg: str = ...) -> None: ...
 
 class IncompleteExpressionError(ParseError):
-    expression: Any
-    lex_position: Any
-    token_type: Any
-    token_value: Any
-    def set_expression(self, expression) -> None: ...
+    expression: str
+    def set_expression(self, expression: str) -> None: ...
 
 class LexerError(ParseError):
-    lexer_position: Any
-    lexer_value: Any
-    message: Any
-    expression: Any
-    def __init__(self, lexer_position, lexer_value, message, expression: Any | None = ...) -> None: ...
+    lexer_position: int
+    lexer_value: str
+    message: str
```

Comment:
These attributes are all the same as in the superclass; no need to override them in the stub

```suggestion
```

Reply:
I don't think it is, no, but I can see how it seemed like it, as the names are similar
lex_position -> lexer_position
token_value -> lexer_value
msg -> message

Reply:
Oh wow, thanks, I completely misread that one. 🤦‍♂️